spacksites/spack-env-templates/slurm/kathleen.yaml

```diff
@@ -29,78 +29,11 @@ spack:
   - /lustre/apps/spack/0.23/deploy/2025-02/spack/var/spack/environments/intel
 
   modules:
-#    # using modulesets to get flat categories
-#    compilers:
-#      roots:
-#        tcl: /shared/ucl/apps/spack/0.23/deploy/2025-02/modules/compilers
-#
-#      tcl:
-#        hide_implicits: true
-#        all:
-#          autoload: direct
-#          # conflict with all other compilers in this section
-#          conflict:
-#            - 'compilers'
-#        include: ['gcc', 'intel-one-api', 'intel']
-#        exclude: ['@:']
-#        hash_length: 0
-#
-#        projections:
-#          all:  'compilers/{name}/{version}/{compiler.name}-{compiler.version}'
-
-#    mpi:
-#      roots:
-#        tcl: /shared/ucl/apps/spack/0.23/deploy/2025-02/modules/mpi
-#
-#      tcl:
-#        hide_implicits: true
-#        all:
-#          autoload: direct
-#          # conflict with all other mpis in this section
-#          conflict:
-#            - 'mpi'
-#        include: ['openmpi', 'intel-mpi', 'intel-oneapi-mpi']
-#        exclude: ['@:']
-#        hash_length: 0
-#
-#        projections:
-#          all:  'mpi/{name}/{version}/{compiler.name}-{compiler.version}'
-
-#    libraries:
-#      roots:
-#        tcl: /shared/ucl/apps/spack/0.23/deploy/2025-02/modules/libraries
-#    
-#      tcl:
-#        hide_implicits: true
-#        all:
-#          autoload: direct
-#          # conflict with everything of same name
-#          conflict:
-#            - '{name}'
-#        include: ['boost', 'cgal', 'eigen', 'fftw', 'gsl', 'gulp', 'hdf5', 'htslib', 'libxc', 'mesa', 'metis', 'mumps',
-#                  'netcdf-c', 'netcdf-fortran', 'netlib-scalapack', 'numactl', 'openblas', 'parmetis', 'quip', 
-#                  'snappy', 'suitesparse', 'udunits', 'vtk']
-#        exclude: ['@:']
-#        hash_length: 0
-#
-#        projections:
-#          fftw +openmp ^mpi: '{name}/{version}-{^mpi.name}-omp/{compiler.name}-{compiler.version}'
-#          openblas threads=openmp: '{name}/{version}-omp/{compiler.name}-{compiler.version}'
-#
-#          ^cuda ^mpi: '{name}/{version}-{^mpi.name}-cuda/{compiler.name}-{compiler.version}'
-#          ^mpi: '{name}/{version}-{^mpi.name}/{compiler.name}-{compiler.version}'
-#          ^cuda: '{name}/{version}-cuda/{compiler.name}-{compiler.version}'
-#          all:  '{name}/{version}/{compiler.name}-{compiler.version}'
-
     default:
       roots:
-#        tcl: /shared/ucl/apps/spack/0.23/deploy/2025-02/modules/applications
         tcl: /lustre/apps/spack/0.23/deploy/2025-02/modules
```

Comment:
I wasn't sure whether this should have been changed to just use the default https://github.com/UCL-ARC/hpc-spack/blob/3a3bb51c2cc312f409eb1de890b4fcf87e9770bd/spacksites/settings/initial_site_modules.yaml#L6 so I just kept the current setting.

Reply:
`/lustre/apps/spack/0.23/deploy/2025-02/modules` is the real location we're creating the deployed modules. It could be changed to `$spack/../modules` in kathleen.yaml since that's the same location.

`$spack/../modules/$env` or similar is what we want environments to use by default otherwise. 

(Although $env comes out at `/lustre/apps/spack/0.23/deploy/2025-02/spack/var/spack/environments/env_name` so we've got all the `/lustre` in the middle of the path when we'd prefer just $env_name - I was considering if we should put it as `tcl: $env/modules` in initial_site_modules.yaml for that default case).